lib/alex_koin/coins/coin.ex

```diff
@@ -22,8 +22,16 @@ defmodule AlexKoin.Coins.Coin do
   end
 
   def for_wallet(wallet, amount) do
-    AlexKoin.Coins.Coin
-    |> Ecto.Query.where(wallet_id: ^wallet.id)
-    |> Ecto.Query.limit(^amount)
+    from c in AlexKoin.Coins.Coin,
```

Comment:
I like this so much more.

You should be able to change `AlexKoin.Coins.Coin` to `__MODULE__`, which is the reference to the current module in this case `Coin`.

Or if you like better, you can `alias __MODULE__` and change this to `Coin`.

Up to you. I personally like the former. Seems more explicit.